### ISSUE TITLE
Roboto font does not cover Chinese and Japanese signs

### ISSUE THREAD

**Issue by Fischaela** [podlove/podlove-subscribe-button#110]
> Roboto supports Latin, Greek (partial) and Cyrillic scripts.[4] On Android, the Noto font is used for languages not covered by Roboto, including Chinese (simplified and traditional), Japanese, Korean, Thai and Hindi.[5]

Source: [Wikipedia](https://en.wikipedia.org/wiki/Roboto)

Use another font, when these languages are selected. Maybe use Noto, as suggested above.

<img width="227" alt="bildschirmfoto 2016-06-21 um 21 36 03" src="https://cloud.githubusercontent.com/assets/3409561/16295399/049523fa-3927-11e6-9e04-4e08cf3e94f0.png">

**Comment by timpritlove** [comment]
Are there any other Open Source Fonts that might cover the all scripts?

**Comment by Fischaela** [comment]
The Noto font (by Google) is the first that comes to my mind in relation to support many scripts and be open source: [https://www.google.com/get/noto/](https://www.google.com/get/noto/). There is also the Source Han Sans font by Adobe: [http://blog.typekit.com/2014/07/15/introducing-source-han-sans/](http://blog.typekit.com/2014/07/15/introducing-source-han-sans/)

**Comment by jperasmus** [comment]
Thanks @Fischaela, this totally helped me. Had a similar issue using Roboto to render Japanese and for some characters it actually rendered Chinese symbols.

**Comment by eteubert** [comment]
I guess this is the same issue as here, where Chinese (zh) is selected as language?

https://jsfiddle.net/twbfjbge/4/

Is there any workaround other than waiting for the fix?

**Comment by jperasmus** [comment]
Yeah, Japanese and Chinese are both affected by this. I got around the problem for both languages using @Fischaela's suggestion and conditionally uses Noto instead of Roboto when language is set to either `ja` or `zh`. I set the `lang="zh"` attribute on the `html` tag and then in my CSS just use something like this:

```
h1 {
  font-family: 'Roboto', sans-serif;
}

[lang="zh"] h1 {
  font-family: 'Noto Sans', sans-serif;
}
```

**Comment by haslinger** [comment]
I wanted to let you know, that the Roboto fonts included don't include the Cyrillic and Greek character sets. I patched them in my project, so you could pick them up from there (or download them from Google, as always): 
https://github.com/jekyll-octopod/jekyll-octopod/tree/master/assets/subscribe-button/fonts

**Comment by timpritlove** [comment]
Is it possible to patch japanesse and chinese fonts in too?

**Comment by haslinger** [comment]
Computer says no ;-) 
Roboto does only come with the character sets mentioned above, see also: https://github.com/google/roboto/issues/228
And patching in characters from a different font is a visual mess.

Typically it's suggested to exchange Roboto for Noto for getting a bigger character set. They are similar, but nowhere near identical, see: [this comparison](http://www.identifont.com/differences?first=roboto&second=Noto+Sans&q=Go).
